lib/membrane/srtp/decryptor.ex

```diff
@@ -40,14 +42,38 @@ defmodule Membrane.SRTP.Decryptor do
     {:ok, %{state | srtp: nil}}
   end
 
+  @impl true
+  def handle_event(_pad, %{handshake_data: handshake_data}, _ctx, state) do
```

Comment:
assert policies are empty

Reply:
I left policies insted of policy because SRTCP uses Encryptor with policies so I am not sure if we want to assert here that they are empty. Do we?